### DISCUSSION TITLE
Save images from clipboard like Obsidian?

### DISCUSSION THREAD

**Discussion by rodhash** [nvim-neorg/neorg#1557] [General]
Something I really enjoyed from Obsidian is this command:
```vim
- `:ObsidianPasteImg [IMGNAME]` to paste an image from the clipboard into the
    note at the cursor position by saving it to the vault and adding a markdown
    image link. You can configure the default folder to save images to with the
    `attachments.img_folder` option.
```

It saves the image right from the system clipboard into a file..

Anything similar here? I couldn't find in the docs

**Comment by max397574** [comment]
ig the best option would be this plugin https://github.com/HakonHarnes/img-clip.nvim which unfortunately doesn't support norg
should be possible to add support for it though

**Reply by hakonharnes** [reply]
Author of img-clip.nvim here. Please open an issue describing the correct markup for embedding images in norg, and I can take a look at it. 

Img-clip supports any markup language (it's configurable per file type), but I can add out-of-the-box support for norg if you'd like.

**Reply by bosvik** [reply]
All you need to do is specify the norg filetype in `img-clip` options
```lua

filetypes = {
      markdown = {
        url_encode_path = true,
        template = "![$FILE_NAME]($FILE_PATH)",
      },
      norg = {
        url_encode_path = true,
        template = ".image $FILE_PATH",
      },
    },
```
